cgmanifests/generated/cgmanifest.json

```diff
@@ -17,8 +17,8 @@
         "Type": "other",
         "other": {
           "Name": "automake",
-          "Version": "1.16.4",
-          "DownloadUrl": "http://ftp.gnu.org/gnu/automake/automake-1.16.4.tar.gz"
+          "Version": "1.16.5",
```

Comment:
>Version": "1.16.5", [](http://example.com/codeflow?start=11&length=19)

just curious,how you generated these changes?

Reply:
3 ways diff and merge. 

1. The files in our repo
2. The files in the upstream repo with the specific commit id which can be found in the README.txt in our build_script dir.
3. The files in the upstream repo with the latest version